### PR TITLE
docs: update overview for embedded accounts

### PR DESCRIPTION
**Pull request by willhennessy**
## PR-Codex overview
This PR enhances the **Account Kit** documentation, emphasizing the benefits of **Embedded Accounts** and improvements in **Modular Account** and **Light Account** descriptions.

### Detailed summary
- **Modular Account** now emphasizes extensibility with custom plugins.
- **Light Account** description highlights key features and deployment addresses.
- Updates in **Embedded Accounts** section showcase the advantages of using **Account Kit** components.
- Improved metadata and content in the **Getting Started** overview.

> The following files were skipped due to too many changes: `site/getting-started/overview.md`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by willhennessy** [comment]
@avasisht23 since I'm heading OOO mon-wed, can you please own reviewing and merging this PR alongside your quickstart? It's good to go from my pov. Thanks!

File: site/public/images/account-kit-overview.jpg
Comment (avasisht23) [inline site/public/images/account-kit-overview.jpg:1]: do we have a better image that this? the "Alchemy Signer" font doesn't match the rest of the cards. I'll ask Kate.

File: site/smart-accounts/index.md [hunk 1]
```diff
@@ -26,24 +26,22 @@ A smart account is an [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) smart
...
 
 ## Light Account
 
 Account Kit provides a default smart account called `LightAccount`.
 
-[Light Account](/smart-accounts/light-account/) is a secure, audited, gas-optimized, ERC-4337-compatible smart account implementation. It has features like ownership transfers, [ERC-1271](https://eips.ethereum.org/EIPS/eip-1271) message signing, and batched transactions. It is also [open](https://github.com/alchemyplatform/light-account) source](https://github.com/alchemyplatform/light-account)!
+[Light Account](/smart-accounts/light-account/) is a secure, audited, gas-optimized, ERC-4337-compatible smart account implementation. It has features like ownership transfers, [ERC-1271](https://eips.ethereum.org/EIPS/eip-1271) message signing, and batched transactions. It is also [open](https://github.com/alchemyplatform/light-account) source](<https://github.com/alchemyplatform/light-account>)!
```
Comment (avasisht23) [inline site/smart-accounts/index.md:41]:
```suggestion
[Light Account](/smart-accounts/light-account/) is a secure, audited, gas-optimized, ERC-4337-compatible smart account implementation. It has features like ownership transfers, [ERC-1271](https://eips.ethereum.org/EIPS/eip-1271) message signing, and batched transactions. It is also [open source](https://github.com/alchemyplatform/light-account)!
```

File: site/smart-accounts/index.md [hunk 2]
```diff
@@ -26,24 +26,22 @@ A smart account is an [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) smart
...
 
-By following the [ERC-6900](https://eips.ethereum.org/EIPS/eip-6900) framework to unlock powerful features for smart accounts as plugins, Modular Account supports supercharging your accounts to provide a fully **customized** and **secure** user experience. We have created two pre-built plugins to level up your smart accounts, [`MultiOwnerPlugin`](/using-smart-accounts/transfer-ownership/modular-account) and [`SessionKeyPlugin`](/using-smart-accounts/session-keys/), and we look forward to what new plugins you create!
+Modular Account is the first [ERC-6900](https://eips.ethereum.org/EIPS/eip-6900) account, making it infinitely extensible with custom plugins. Plug and play from a selection of existing plugins, including session keys and account recovery, or write your own to customize the account for your app. We have created two pre-built plugins available today, [`MultiOwnerPlugin`](/using-smart-accounts/transfer-ownership/modular-account) and [`SessionKeyPlugin`](/using-smart-accounts/session-keys/). We're actively building new plugins such as multisig, and we look forward to what new plugins you create!
```
Comment (avasisht23) [inline site/smart-accounts/index.md:31]:
```suggestion
Modular Account is the first [ERC-6900](https://eips.ethereum.org/EIPS/eip-6900) account implementation, making it infinitely extensible with custom plugins. Plug and play from a selection of existing plugins, including session keys and account recovery, or write your own to customize the account for your app. We have created two pre-built plugins available today, [`MultiOwnerPlugin`](/using-smart-accounts/transfer-ownership/modular-account) and [`SessionKeyPlugin`](/using-smart-accounts/session-keys/). We're actively building new plugins such as multisig, and we look forward to what new plugins you create!
```

File: site/getting-started/overview.md [hunk 3]
```diff
@@ -18,79 +18,80 @@ head:
...
 
-- **Familiar, secure login:** Streamline your sign-up flow with custom [authentication methods](/signers/choosing-a-signer), including email, social login, SMS, biometrics, passkeys, or self-custodial signers like Metamask.
-- **Sponsor gas:** enable users to transact without gas by [sponsoring gas](/using-smart-accounts/sponsoring-gas/gas-manager) fees through an ERC-4337 paymaster.
-- **Batch transactions:** streamline multi-step flows into a single transaction by [batching actions](/using-smart-accounts/batch-user-operations) in a UserOp to the Bundler API.
-- **Delightful developer experience:** Account Kit is built on Viem and fully compatible with ethers.js and the EIP-1193 interface to make integration a breeze.
+# How it works
 
-# What is Account Kit?
+**Embedded Accounts** use [ERC-4337 Account Abstraction](https://www.youtube.com/watch?v=Vpk_MhY-EeE) to build next-gen wallet UX. Account Abstraction dramatically simplifies the user experience by adding a few new developer components like a smart account, bundler, and paymaster. You can also configure your embedded accounts as a lightweight EOA if you prefer.
```
Comment (avasisht23) [inline site/getting-started/overview.md:37]: for the last sentence, maybe we link out the dev docs: https://accountkit.alchemy.com/packages/aa-alchemy/signer/overview.html#using-the-signer-as-an-eoa

File: site/getting-started/overview.md [hunk 4]
```diff
@@ -18,79 +18,80 @@ head:
...
 
-**Account Kit** provides all the tools you need to build smart accounts in your web3 app, unlocking powerful features like social login, gas sponsorship, and batched transactions. Integrate [account abstraction](https://www.alchemy.com/overviews/what-is-account-abstraction/?a=ak-docs) with just a few lines of code to deploy your first smart accounts and start sending user operations!
+Account Kit provide all the tools you need to onboard every user to an invisible wallet. The toolkit includes:
```
Comment (avasisht23) [inline site/getting-started/overview.md:39]:
```suggestion
Account Kit provides all the tools you need to onboard every user to an invisible wallet. The toolkit includes:
```

File: site/getting-started/overview.md [hunk 5]
```diff
@@ -18,79 +18,80 @@ head:
...
 Let’s dive into each component.
 
 ## aa-sdk
 
-The `aa-sdk` is a type-safe and performant TypeScript library built on top of [viem](https://viem.sh/) to provide ergonomic methods for sending user operations, sponsoring gas, and deploying smart accounts. It handles all the complexity of ERC-4337 under the hood to simplify account abstraction.
+The `aa-sdk` is a type-safe and performant TypeScript library built on top of [viem](https://viem.sh/) and fully compatible with ethers.js. It provides ergonomic methods for sending user operations, sponsoring gas, deploying smart accounts, and much more. It handles all the complexity of ERC-4337 under the hood to simplify your development.
+
+The SDK implements an EIP-1193 provider interface so embedded accounts can drop right into popular app frameworks and wallet connect libraries like RainbowKit, Wagmi, and Web3Modal. It also includes ethers.js adapters to provide full support for ethers.js apps.
+
+Get started with the [quick start](/getting-started/setup) guide or browse the open source [aa-sdk github](https://github.com/alchemyplatform/aa-sdk).
```
Comment (avasisht23) [inline site/getting-started/overview.md:57]: new quickstart is at this link
```suggestion
Get started with the [quick start](/getting-started/introduction) guide or browse the open source [aa-sdk github](https://github.com/alchemyplatform/aa-sdk).
```

File: site/getting-started/overview.md [hunk 6]
```diff
@@ -18,79 +18,80 @@ head:
...
 We built our [Bundler in Rust](https://www.alchemy.com/blog/open-sourcing-rundler/?a=ak-docs) to handle the highest loads at production scale. It’s able to handle massive scale because we operate it alongside our fleet of nodes powering the biggest dapps in web3 from Opensea to Circle.
 
-Check out the open source code in our affectionately named [Rundler github repo](https://github.com/alchemyplatform/rundler).
+Check out the open source code in our affectionately named [Rundler github repo](https://github.com/alchemyplatform/rundler) and start [sending UserOps](/using-smart-accounts/send-user-operations) today.
 
-# Start building with Account Kit
+# Start building
 
-Account Kit was designed from the ground up to make account abstraction easy. It’s built on top of industry-leading infrastructure that powers applications at massive scale from Opensea to Shopify.
+Next, complete the [quick start](/getting-started/setup) guide to make wallets invisible in your onboarding and checkout flows.
```
Comment (avasisht23) [inline site/getting-started/overview.md:95]:
```suggestion
Next, complete the [quick start](/getting-started/introduction) guide to make wallets invisible in your onboarding and checkout flows.
```

File: site/getting-started/overview.md [hunk 7]
```diff
@@ -18,79 +18,80 @@ head:
       content: Learn what Account Kit is, how it works, and how you can use it to integrate smart accounts in your app.
 ---
 
-# Why Account Kit?
+# Why Embedded Accounts?
```
Comment (avasisht23) [inline site/getting-started/overview.md:21]: it's a bit confusing for the Account Kit docs' overview to have the first line be about Embedded Accounts with no mention of how its powered by Account Kit.

File: site/getting-started/overview.md
Comment (avasisht23) [inline site/getting-started/overview.md:1]: we should update these meta tags if we're making this overview more about embedded accounts

File: site/getting-started/overview.md (hunk 3 above)
Comment (willhennessy) [reply site/getting-started/overview.md:37]: ok added

File: site/getting-started/overview.md
Comment (willhennessy) [reply site/getting-started/overview.md:1]: ok

**Review by moldy530 — APPROVED** [review]

**Pull request merged**